BDArmory/UI/BDAEditorArmorWindow.cs

```diff
@@ -331,9 +331,9 @@ void WindowArmor(int windowID)
                 if (!FerramAerospace.hasFAR)
                 {
                     line++;
-                    GUI.Label(new Rect(10, line * lineHeight, 300, lineHeight), StringUtils.Localize("#LOC_BDArmory_ArmorTotalLift") + ": " + totalLift.ToString("0.00"), style);
+                    GUI.Label(new Rect(10, line * lineHeight, 300, lineHeight), StringUtils.Localize("#LOC_BDArmory_ArmorTotalLift") + ": " + totalLift.ToString("0.00" + " m2"), style);
```

Comment:
String concatenation with `+` causes more GC allocation than using interpolated strings (`$"..."`), so prefer interpolated strings or StringBuilder (for larger multi-line stuff) when possible, especially for GUI stuff.
So
```csharp
$"{StringUtils.Localize("#LOC_BDArmory_ArmorTotalLift")}: {totalLift:0.00} m2"
```

Reply:
Good point. Fixed.
I'm sure the code is littered with string concatenation in debug messages and the like, will have to go through at somepoint and find them all

Reply:
Yeah, it's not so important if it's not in something that's being used frequently, but it was quite significant in the profiler for debug labels and text in the various GUIs.